### PR TITLE
[Composer] Reveal Triggerer to GA.

### PR DESCRIPTION
**Pull request by apilaskowski**
Google Cloud Composer is releasing Triggerer to GA.
Fixes [16024](https://github.com/hashicorp/terraform-provider-google/issues/16024)

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
composer: promoted `config.0.workloads_config.0.triggerer` to GA in resource `google_composer_environment`
```

### CODE REVIEW

**Comment by zli82016** [comment]
Feel free to let me know when it is ready for review.

**Comment by apilaskowski** [comment]
@zli82016 Hi, we (Composer) are just releasing Triggerer to GA, so we should wait a few days, so api is revealed and everything works properly on our side.

Additionally, I have some problems trying to run acceptance tests locally, and I asked @rlapin-pl to help me with that.

**Comment by apilaskowski** [comment]
@zli82016 please let me know if everything is ok.
Can I do sth to fix one failing presubmit?

File: mmv1/Gemfile.lock [hunk 1]
```diff
@@ -6,6 +6,7 @@ GEM
       i18n (>= 1.6, < 2)
       minitest (>= 5.1)
       tzinfo (~> 2.0)
+      zeitwerk (~> 2.3)
```
Comment (zli82016) [inline mmv1/Gemfile.lock:9]: What is the reason to add `zeitwerk`?

Comment (apilaskowski) [reply mmv1/Gemfile.lock]: It was here, because of my misconfiguration. Removed, thanks for noticing!

**Comment by zli82016** [comment]
Hello, @apilaskowski , have you run the tests including `triggerer` in terraform google provider GA locally?

**Comment by apilaskowski** [comment]
> Hello, @apilaskowski , have you run the tests including `triggerer` in terraform google provider GA locally?

I did execute command:
```
$ make testacc TEST=./google/services/composer/ TESTARGS='-run=TestAccComposerEnvironment_withDatabaseConfig'
```

And it succeeded for me.

**Comment by apilaskowski** [comment]
My bad, thanks @rlapin-pl for pointing it out for me. Now, I executed:

```
 make testacc TEST=./google/services/composer/ TESTARGS='-run=TestAccComposerEnvironment'
 ```

I will confirm the status when tests are done.

**Comment by apilaskowski** [comment]
> My bad, thanks @rlapin-pl for pointing it out for me. Now, I executed:
> 
> ```
>  make testacc TEST=./google/services/composer/ TESTARGS='-run=TestAccComposerEnvironment'
> ```
> 
> I will confirm the status when tests are done.

For me they passed. 

What about ```VCR-test```, i don't see any details there.

**Comment by apilaskowski** [comment]
> Tests passed during RECORDING mode: `TestAccAlloydbUser_updatePassword_BuiltIn`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbUser_updatePassword_BuiltIn.log)] `TestAccAlloydbBackup_alloydbBackupFullExample`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbBackup_alloydbBackupFullExample.log)] `TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButMissingClusterTypeSecondary`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButMissingClusterTypeSecondary.log)] `TestAccAlloydbCluster_secondaryClusterWithNetworkConfigAndAllocatedIPRange`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbCluster_secondaryClusterWithNetworkConfigAndAllocatedIPRange.log)] `TestAccAlloydbBackup_createBackupWithMandatoryFields`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbBackup_createBackupWithMandatoryFields.log)] `TestAccAlloydbUser_updateRoles_IAM`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbUser_updateRoles_IAM.log)] `TestAccAlloydbUser_alloydbUserBuiltinExample`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbUser_alloydbUserBuiltinExample.log)] `TestAccAlloydbCluster_secondaryClusterMandatoryFields`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbCluster_secondaryClusterMandatoryFields.log)] `TestAccAlloydbCluster_secondaryClusterUpdate`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbCluster_secondaryClusterUpdate.log)] `TestAccAlloydbCluster_secondaryClusterUsingCMEK`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbCluster_secondaryClusterUsingCMEK.log)] `TestAccAlloydbInstance_createPrimaryAndReadPoolInstance`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbInstance_createPrimaryAndReadPoolInstance.log)] `TestAccAlloydbInstance_createInstanceWithMandatoryFields`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbInstance_createInstanceWithMandatoryFields.log)] `TestAccAlloydbInstance_createInstanceWithNetworkConfigAndAllocatedIPRange`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbInstance_createInstanceWithNetworkConfigAndAllocatedIPRange.log)] `TestAccAlloydbInstance_update`[[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbInstance_update.log)]
> 
> ##### Rerun these tests in REPLAYING mode to catch issues
> No issues found for passed tests after REPLAYING rerun.
> 
> Tests failed during RECORDING mode: `TestAccAlloydbInstance_clientConnectionConfig_update`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/build-log/recording_build/TestAccAlloydbInstance_clientConnectionConfig_update_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbInstance_clientConnectionConfig_update.log)] `TestAccAlloydbBackup_alloydbBackupBasicExample`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/build-log/recording_build/TestAccAlloydbBackup_alloydbBackupBasicExample_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbBackup_alloydbBackupBasicExample.log)] `TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButClusterTypeIsPrimary`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/build-log/recording_build/TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButClusterTypeIsPrimary_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbCluster_secondaryClusterDefinedSecondaryConfigButClusterTypeIsPrimary.log)] `TestAccAlloydbCluster_secondaryClusterMissingSecondaryConfig`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/build-log/recording_build/TestAccAlloydbCluster_secondaryClusterMissingSecondaryConfig_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbCluster_secondaryClusterMissingSecondaryConfig.log)] `TestAccAlloydbBackup_update`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/build-log/recording_build/TestAccAlloydbBackup_update_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbBackup_update.log)] `TestAccAlloydbCluster_restore`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/build-log/recording_build/TestAccAlloydbCluster_restore_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbCluster_restore.log)] `TestAccAlloydbCluster_secondaryClusterWithNetworkConfig`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/build-log/recording_build/TestAccAlloydbCluster_secondaryClusterWithNetworkConfig_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbCluster_secondaryClusterWithNetworkConfig.log)] `TestAccAlloydbUser_alloydbUserIamExample`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/build-log/recording_build/TestAccAlloydbUser_alloydbUserIamExample_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbUser_alloydbUserIamExample.log)] `TestAccAlloydbUser_updateRoles_BuiltIn`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/build-log/recording_build/TestAccAlloydbUser_updateRoles_BuiltIn_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbUser_updateRoles_BuiltIn.log)] `TestAccAlloydbCluster_secondaryClusterAddAutomatedBackupPolicy`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/build-log/recording_build/TestAccAlloydbCluster_secondaryClusterAddAutomatedBackupPolicy_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbCluster_secondaryClusterAddAutomatedBackupPolicy.log)] `TestAccAlloydbBackup_usingCMEK`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/build-log/recording_build/TestAccAlloydbBackup_usingCMEK_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbBackup_usingCMEK.log)] `TestAccAlloydbInstance_clientConnectionConfig_sslModeDefault`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/build-log/recording_build/TestAccAlloydbInstance_clientConnectionConfig_sslModeDefault_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbInstance_clientConnectionConfig_sslModeDefault.log)] `TestAccAlloydbCluster_alloydbSecondaryClusterBasicExample`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/build-log/recording_build/TestAccAlloydbCluster_alloydbSecondaryClusterBasicExample_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbCluster_alloydbSecondaryClusterBasicExample.log)] `TestAccAlloydbInstance_alloydbInstanceBasicExample`[[Error message](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/build-log/recording_build/TestAccAlloydbInstance_alloydbInstanceBasicExample_recording_test.log)] [[Debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording/TestAccAlloydbInstance_alloydbInstanceBasicExample.log)]
> 
> Please fix these to complete your PR. View the [build log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/build-log/recording_test.log) or the [debug log](https://console.cloud.google.com/storage/browser/ci-vcr-logs/beta/refs/heads/auto-pr-9088/artifacts/bb83ab16-1204-44a2-90c9-99fd7d1336fb/recording) for each test

Is alloydb another product, have I touched it, why are those tests influenced?

**Comment by zli82016** [comment]
The alloydb tests are unrelated. Sorry for the confusion. Thanks for running the tests locally.

**Comment by apilaskowski** [comment]
@zli82016 let me know if it can be merged, or we should do sth more.
When this will be merged what is the timeline for having it released for customers?

**Review by zli82016 — APPROVED** [review]
LGTM

**Comment by zli82016** [comment]
> @zli82016 let me know if it can be merged, or we should do sth more. When this will be merged what is the timeline for having it released for customers?

Yeah, it looks good and merged. Thanks for working on it. It is expected to be released to the customers in 5.3.0 on Oct 23th.

**Pull request merged**